include/inexor/vulkan-renderer/world/cube.hpp

```diff
@@ -2,9 +2,11 @@
 
 #include "inexor/vulkan-renderer/world/indentation.hpp"
 
+#include <glm/geometric.hpp>
 #include <glm/vec3.hpp>
 
 #include <array>
+#include <cmath>
```

Comment:
Instead of including `cmath`, there's also equivalent glm functions, e.g. `glm::pow`

Reply:
I will use `glm::pow`. Done.